### PR TITLE
Adding rule encrypt-data-using-rc4-with-custom-key-via-winapi.yml

### PR DESCRIPTION
**Pull request by MalwareMechanic**
The standard method of using RC4 encryption via the Windows WinCrypt APIs involves generating and importing two key BLOBs:

- **PRIVATEKEYBLOB**
- **SIMPLEBLOB**

The **SIMPLEBLOB** structure contains the encrypted form of the generated RC4 key (i.e., the RC4 key data is not in plaintext). During typical RC4 key generation, Windows does not allow the developer to choose their own RC4 key; however, selecting a custom RC4 key is possible as outlined in this blog post: https://www.phdcc.com/cryptorc4.htm.

By utilizing the structures **PrivateKeyWithExponentOfOne** and **SimpleBlobRC4KeyTemplate**, a developer can set a custom RC4 key by overwriting 16 bytes of **SimpleBlobRC4KeyTemplate** with their key in reverse order starting at offset 0x0C.

This rule matches this technique.

### CODE REVIEW

File: data-manipulation/encryption/rc4/encrypt-data-using-rc4-with-custom-key-via-winapi.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    name: encrypt data using RC4 with custom key via WinAPI
+    namespace: data-manipulation/encryption/rc4
+    author: blaine.stancill@mandiant.com
+    scope: function
+    att&ck:
+      - Defense Evasion::Obfuscated Files or Information [T1027]
+    mbc:
+      - Defense Evasion::Obfuscated Files or Information::Encryption-Standard Algorithm [E1027.m05]
+      - Cryptography::Encrypt Data::RC4 [C0027.009]
+    references: https://www.phdcc.com/cryptorc4.htm
```
Comment (williballenthin) [inline data-manipulation/encryption/rc4/encrypt-data-using-rc4-with-custom-key-via-winapi.yml:12]: i think this should be a list
```suggestion
    references: 
      - https://www.phdcc.com/cryptorc4.htm
```

File: data-manipulation/encryption/rc4/encrypt-data-using-rc4-with-custom-key-via-winapi.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    examples:
+      - 4E9C546A54E40D0DA89BB4616DD7F8C4:0x140007B70
+      - A563C50C5FA0FD541248ACAF72CC4E7D:0x401AF0
+  features:
+    - and:
+      - api: CryptImportKey
+      - number: 0x4C = SimpleBlobRC4KeyTemplate size
+      - bytes: 01 02 00 00 01 68 00 00 00 A4 00 00 = SimpleBlobRC4KeyTemplate header
```
Comment (williballenthin) [inline data-manipulation/encryption/rc4/encrypt-data-using-rc4-with-custom-key-via-winapi.yml:20]: these bytes with comment are really cool

**Review by williballenthin — APPROVED** [review]
thank you!

**Comment by williballenthin** [comment]
linter issues:

```
  FAIL: EOF format: end file with a single empty line
  FAIL: rule format incorrect: use scripts/capafmt.py or adjust as follows
```

**Review by mr-tz — APPROVED** [review]
nice!

**Pull request merged**